Source/Android/jni/ButtonManager.cpp

```diff
@@ -586,13 +586,19 @@ void Init(const std::string& gameId)
       {
         hasbind = true;
         type = BIND_AXIS;
-        sscanf(value.c_str(), "Device '%127[^\']'-Axis %d%c", dev, &bindnum, &modifier);
+        if (std::string::npos != value.find("Device ''"))
```

Comment:
Is there any particular reason why you're using `find` instead of `StringBeginsWith` (available in Common/StringUtil.h)?

Reply:
Changed. I will have to retest the patch